plugins/renepay/heap.h

```diff
@@ -0,0 +1,195 @@
+#ifndef LIGHTNING_PLUGINS_RENEPAY_HEAP_H
+#define LIGHTNING_PLUGINS_RENEPAY_HEAP_H
+#include "config.h"
+#include <ccan/tal/tal.h>
+#include <gheap.h>
+#include <stdint.h>
```

Comment:
`heap.h` is not being used. It was originally meant to add a search function `gheap_upper_bound` missing in `gheap` that would allow us to remove the hack of the global context for `dijsktra`. Still this comment is valid for `dijsktra.c`, `#define NDEBUG 1` should be added to remove the internal checks of `gheap`.

Reply:
Ah, just delete it then, thanks! Same with any other unused files.

Reply:
Done https://github.com/ElementsProject/lightning/pull/6376/commits/92689dfaa6dff322d9da4a63d1450b67f3eb6bf7